src/main/java/se/sundsvall/billingpreprocessor/integration/db/BillingRecordRepository.java

```diff
@@ -1,11 +1,9 @@
 package se.sundsvall.billingpreprocessor.integration.db;
 
-import javax.transaction.Transactional;
-
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+import jakarta.transaction.Transactional;
```

Comment:
Isn't `@org.springframework.transaction.annotation.Transactional` preferred in SpringBoot-applications?